docs/mkDocs/docs/options.md

```diff
@@ -80,57 +80,6 @@ instantiation of polymorphic type variables.
 It is suggested to use on theorem proving style when reflected 
 functions are trivially refined. 
 
-## Incremental Checking
```

Comment:
Can we add this back as it may be useful even with the new plugin mode. (It focuses the check on _binders_ that have been modified which can be much quicker for large files during interactive development.)